RxSwift/Subjects/Variable.swift

```diff
@@ -10,7 +10,7 @@
 ///
 /// Unlike `BehaviorSubject` it can't terminate with error, and when variable is deallocated
 /// it will complete it's observable sequence (`asObservable`).
-public final class Variable<Element> {
+public final class Variable<Element>: ObservableConvertibleType {
```

Comment:
Hi there, I'm interested that why `Variable` does not conform to `ObservableConvertibleType` and `ObserverType`.

In my opinion, I think that the type `A` that implemented `ObservableConvertibleType` make `B` as `Observable` and some other object should retain B object, not A object directly.
Then `Variable` case, the `Observable` is a internal `BehaviorSubject` so, `Variable` never be retained other object. 

But now, maybe some of Observable family has `ObservableConvertibleType` directly. Can we change this?